rep-0149.rst

```diff
@@ -178,55 +177,30 @@ Therefore the need to avoid unnecessary rebuilds has increased.
 It is also desired to be able to encourage more frequent releases if they
 don't require downstream packages to be rebuilt.
 
-Dependencies across ROS 1 and ROS 2
+Packages across ROS 1 and ROS 2
 -----------------------------------
 
-The differences in dependencies in ROS 1 and ROS 2 could be handled in various
-ways.
-
-Use the same package names
-''''''''''''''''''''''''''
-
-One option (A) would be to not allow different dependencies depending on the
-ROS version.
-That would imply that for the previously described example of `roscpp` and
-`rclcpp` a "dummy" package could be provided to provide the same package names.
-This could be either a ROS 1 package named `rclcpp` which simply depends on
-`roscpp` or a ROS 2 package named `roscpp` which depends on `rclcpp`.
-That would allow all downstream packages to use a single name as their
-dependency.
-While that would be possible it would require dummy packages for every naming
-difference between ROS 1 and ROS 2.
-
-But option (A) would become even more cumbersome if a package needs to declare
-a dependency only in one of the ROS versions.
-It would require a dummy package in both ROS version where as in one of them
-the dummy would be empty and in the other it would declare a dependency.
```

Comment:
Imo this alternative approach should not be removed.

Reply:
I was assuming listing alternatives wasn't required once an option is picked. How about this section becomes an explanation of why it wasn't chosen?

Reply:
Sure, I just don't think removing this valuable information all together is good. "Refactoring" is fine.

Reply:
Added alternatives to appendix in eec65c1e8c8286235461f2054ff06909d70dc60b